api/Dockerfile

```diff
@@ -15,10 +15,14 @@ RUN ./gradlew help
 
 # Copy code and build artefact
 COPY build.gradle.kts /opt/code/build.gradle.kts
+
+# Download dependencies in order to save that build time later
+RUN ./gradlew build
```

Comment:
This step means that changes to `src/` don't then require Gradle to reinstall remote dependencies, which is a thing I believe it was doing. Often cached, but this should still sometimes save 20-30 seconds from a full `--build`.